doc/en/user/source/installation/win_installer.rst

```diff
@@ -1 +1 @@

   .. note:: For more information about Java and GeoServer, please see the section on :ref:`production_java`.
   
   .. figure:: images/win_jre.png

      Selecting a valid JRE

#. Enter the path to your GeoServer data directory or select the default. If this is your first time using GeoServer, select the :guilabel:`Default data directory`. When finished, click :guilabel:`Next`.

   .. figure:: images/win_datadir.png

      Setting a GeoServer data directory

#. Enter the username and password for administration of GeoServer. GeoServer's :ref:`web_admin` requires authentication for management, and what is entered here will become those administrator credentials.  The defaults are :guilabel:`admin / geoserver`.  It is recommended to change these from the defaults. When finished, click :guilabel:`Next`.

   .. figure:: images/win_creds.png

      Setting the username and password for GeoServer administration

#. Enter the port that GeoServer will respond on. This affects the location of the GeoServer :ref:`web_admin`, as well as the endpoints of the GeoServer services such as :ref:`wms` and :ref:`wfs`.  The default port is :guilabel:`8080`, though any valid and unused port will work. When finished, click :guilabel:`Next`.

   .. figure:: images/win_port.png

      Setting the GeoServer port

#. Select whether GeoServer should be run manually or installed as a service. When run manually, GeoServer is run like a standard application under the current user. When installed as a service, GeoServer is integrated into Windows Services, and thus is easier to administer. If running on a server, or to manage GeoServer as a service, select :guilabel:`Install as a service`. Otherwise, select :guilabel:`Run manually`.  When finished, click :guilabel:`Next`.

   .. figure:: images/win_service.png

      Installing GeoServer as a service

#. Review your selections and click the :guilabel:`Back` button if any changes need to be made.  Otherwise, click :guilabel:`Install`.

   .. figure:: images/win_review.png

      Verifying settings

#. GeoServer will install on your system.  When finished, click :guilabel:`Finish` to close the installer.

#. If you installed GeoServer as a service, it is already running.  Otherwise, you can start GeoServer by going to the Start Menu, and clicking :guilabel:`Start GeoServer` in the GeoServer folder.

#. Navigate to ``http://localhost:8080/geoserver`` (or wherever you installed GeoServer) to access the GeoServer :ref:`web_admin`.

If you see the GeoServer logo, then GeoServer is successfully installed.

   .. figure:: images/success.png

      GeoServer installed and running successfully

Uninstallation
--------------

GeoServer can be uninstalled in two ways: by running the :file:`uninstall.exe` file in the directory where GeoServer was installed, or by standard Windows program removal.

```

Comment:
What is this? :-D

Reply:
These seem to be Windows Style CRLF, which have been there since the first commit.

Created a pull request to commit these separately:
https://github.com/geoserver/geoserver/pull/3785

I'll try merge those commits into my repo, so the actual changes are more clearly.

Reply:
Deleted pull https://github.com/geoserver/geoserver/pull/3785 as changes have been made directly to this branch